packages/nextjs/package.cjs.json

```
@@ -4,6 +4,13 @@
     "#components": {
       "react-server": "./components.server.js",
       "default": "./components.client.js"
+    },
+    "#safe-node-apis": {
+      "edge-light": "./runtime/browser/safe-node-apis.js",
+      "worker": "./runtime/browser/safe-node-apis.js",
+      "browser": "./runtime/browser/safe-node-apis.js",
+      "node": "./runtime/node/safe-node-apis.js",
+      "default": "./runtime/browser/safe-node-apis.js"
```

Can we just simplify this to ? 
```suggestion
      "node": "./runtime/node/safe-node-apis.js",
      "default": "./runtime/browser/safe-node-apis.js"
```

---

yes, seems safe to do so.